src/Feature.hs

```diff
@@ -13,11 +13,12 @@ import Text.Pandoc.Definition
 import Data.List
 
 data Features = Features { _features :: [Feature]}
-  deriving (Show, Eq)
+  deriving (Show, Eq, Generic)
```

Comment:
what's the Generic instance for?

Reply:
It is needed to infer the `ToJSON` instances, but maybe it's not superfluous now due to explicit `toJSON` override